variables.tf

```diff
@@ -148,3 +148,17 @@ variable "sse_algorithm" {
   description = "The server-side encryption algorithm to use"
   default     = "aws:kms"
 }
+
+variable "sns_arn" {
+  type        = string
+  description = "The arn for the SNS"
+  default     = "topic_arn"
+}
+
+variable "event" {
+  type        = string
+  description = "The event taking place"
+  default     = ""
```

Comment:
What behaviour do you expect when passing an empty string as a default? This is not going to work I believe

Reply:
It's been tested and it does because of the other conditional we've put in place